commit d2b15c8e0e570cc9a3f6611d346ac505cf210719 (refs/changes/13/5113/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-11-22T18:32:37-08:00 (11 months ago)
    
    MANTA-4030 many images using node 0.10 or 0.12 broken by transitive lru-cache@4.1.4 dep
    Reviewed by: Josh Wilsdon <josh@wilsdon.ca>
    Approved by: Josh Wilsdon <josh@wilsdon.ca>

diff --git a/package.json b/package.json
index 61ffa4a..4ac48f4 100644
--- a/package.json
+++ b/package.json
@@ -1,10 +1,11 @@
 {
     "name": "manta-deployment",
     "description": "Manta deployment configuration",
-    "version": "0.0.1",
+    "version": "0.0.2",
     "author": "Joyent (joyent.com)",
     "license": "MPL-2.0",
     "private": true,
+    "// lru-cache": "attempting to pin version while satisfying transitive deps, because lru-cache@4.1.4 includes yallist@3 which breaks node --abort-on-uncaught-exception with node <v4",
     "dependencies": {
         "assert-plus": "0.2.0",
         "async": "0.1.22",
@@ -18,6 +19,7 @@
         "imgapi-cli": "git+https://github.com/joyent/sdc-imgapi-cli.git#65bba66818",
         "jsprim": "^1.4.0",
         "js-yaml": "3.8.2",
+        "lru-cache": "4.1.3",
         "node-uuid": "1.4.0",
         "optimist": "0.3.5",
         "once": "1.3.0",
